sycl/test-e2e/AmdNvidiaJIT/kernel_and_bundle.cpp

```diff
@@ -0,0 +1,110 @@
+// REQUIRES: cuda || hip
+
+// RUN: %{build} -fsycl-embed-ir -o %t.out
+// RUN: env SYCL_JIT_AMDGCN_PTX_KERNELS=1 env SYCL_JIT_COMPILER_DEBUG="sycl-spec-const-materializer" %{run} %t.out &> %t.txt ; FileCheck %s --input-file %t.txt
+
+#include <sycl/sycl.hpp>
```

Comment:
`sycl/sycl.hpp` is not permitted in E2E tests. You want `sycl/detail/core.hpp` and then probably some other headers.

Reply:
Done.